docker/bootstrap/README.md

```diff
@@ -7,6 +7,7 @@ The `vitess/bootstrap` image comes in different flavors:
 
 * `vitess/bootstrap:common`    - dependencies that are common to all flavors
 * `vitess/bootstrap:mysql80`   - bootstrap image for MySQL 8.0
+* `vitess/bootstrap:mysql84`   - bootstrap image for MySQL 8.4
```

Comment:
Should we also have percona84?

Reply:
AFAIK no one in the community has asked for this yet. We should see who is using the percona tags and if they need a 84 tag.